xds/internal/balancer/ringhash/ring.go

```diff
@@ -116,30 +116,37 @@ func newRing(subConns *resolver.AddressMap, minRingSize, maxRingSize uint64, log
 	return &ring{items: items}
 }
 
-// normalizeWeights divides all the weights by the sum, so that the total weight
-// is 1.
+// normalizeWeights calculates the normalized weights for each subConn in the
+// given subConns map. It returns a slice of subConnWithWeight structs, where
+// each struct contains a subConn and its corresponding weight. The function
+// also returns the minimum weight among all subConns.
+//
+// The weight of each subConn is calculated by dividing its weight attribute by
```

Comment:
Nit: s/The weight of each subConn/The normalized weight of each subConn/

Reply:
Makes sense. Done.